src/Axiom/Set/Map.agda

```diff
@@ -146,7 +146,7 @@ filterKeys : {P : A → Type} → specProperty P → Map A B → Map A B
 filterKeys sp-P = filterᵐ (sp-∘ sp-P proj₁)
 
 singletonᵐ : A → B → Map A B
-singletonᵐ a b = ❴ (a , b) ❵
+singletonᵐ a b = singleton (a , b)
```

Comment:
The goal of this PR is to rename the curly braces used for singletons using a type class (called `HasSingleton`).  However, in the `.agda` files that are used by that type class, we will not use the undecorated curley braces because that would cause a cyclic dependency.  So we use `singleton` for what previously was called `❴_❵`.